### PR TITLE
Fix #5931 Can not import a correct zip of csv/tsv files if a directory is zipped instead of only the files

### LINKED ISSUE #5931: Can not import a correct zip of csv/tsv files if a directory is zipped instead of only the files
### How to Reproduce
Import [csv_import.zip](https://github.com/molgenis/molgenis/files/841737/csv_import.zip)
OR
import [tsv_import.zip](https://github.com/molgenis/molgenis/files/841738/tsv_import.zip)

### Expected behavior
Import succeeds, zip contains correct metadata and data. EMX version uploads succesfully

### Observed behavior
```
Import failed for action [null]
java.lang.ArrayIndexOutOfBoundsException: -1
```
Because a directory was zipped, CsvRepositoryCollection parses 7 entity names instead of 6 in the loadEntityNames() method:

1) ""
2) "attributes"
2) "entities"
2) "packages"
2) "root_data"
2) "root_child_reference#1"
2) "root_reference#2"

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits

### CODE REVIEW

File: molgenis-data-csv/src/test/java/org/molgenis/data/csv/CsvIteratorTest.java [hunk 1]
```diff
@@ -56,4 +59,41 @@ public void testIterator() throws IOException
...
+		File zipFile = getZipWithCsvFile("folder/testdata.csv");
+
+		CsvIterator it = new CsvIterator(zipFile, "testdata", null, null, entityType);
+		assertEquals(it.getColNamesMap().keySet(), Sets.newLinkedHashSet(Arrays.asList("col1", "col2")));
+		assertEquals(Iterators.size(it), 5);
+	}
+
+	private File getZipWithCsvFile(String filePath) throws IOException
+	{
+		InputStream in = getClass().getResourceAsStream("/testdata.csv");
```
Comment (dennishendriksen) [inline molgenis-data-csv/src/test/java/org/molgenis/data/csv/CsvIteratorTest.java]: Use ResourceUtils to fix your test failures

**Comment by fdlk** [comment]
Test this please

**Comment by fdlk** [comment]
test this please

**Comment by fdlk** [comment]
test this please

**Comment by fdlk** [comment]
Works great!

**Pull request merged**